### ISSUE TITLE
Can't run stand alone example

### ISSUE THREAD

**Issue by AnotherStop** [uber/deck.gl#929]
I copy stand alone examples (in my case is jax-layer) to another new foler and follow steps in this doc (https://github.com/uber/deck.gl/blob/master/docs/get-started/getting-started.md) to run it.

1. `npm install --save deck.gl luma.gl`
2. `npm install`
3. `export MapboxAccessToken={My Mapbox Token Here}`
4. `npm start`

Then I got a lot of errors like 

```ERROR in ./~/react-map-gl/dist/utils/event-manager/move-input.js
Module not found: Error: Can't resolve 'babel-runtime/core-js/object/assign' in '/Users/myusername/Code/jsx-layers/node_modules/react-map-gl/dist/utils/event-manager'
 @ ./~/react-map-gl/dist/utils/event-manager/move-input.js 7:14-60
 @ ./~/react-map-gl/dist/utils/event-manager/event-manager.js
 @ ./~/react-map-gl/dist/index.js
 @ ./app.js
 @ multi (webpack)-dev-server/client?http://localhost:3000 webpack/hot/dev-server ./app.js
```
Did i miss some steps?

P.S. If I go to the root folder of deck.gl and run `npm install` and go to `examples/layer-browser` run `npm install` and go back to root folder run `npm start`, all examples work well.

**Comment by ibgreen** [comment]
@AnotherStop 

Confirming the issue. `npm install babel-runtime` in the example will fix it.

We should look into using `babel-transform-runtime` to ensure this is not needed.

**Comment by AnotherStop** [comment]
@ibgreen Thanks for your reply! `npm install babel-runtime` fixed it. Webpack compiled successfully but when I go to `localhost:3000`, it displays 

![screen shot 2017-09-13 at 8 04 45 am](https://user-images.githubusercontent.com/4811429/30376496-98cdeb0e-985a-11e7-966c-5cff8068426d.png)
 
and warnings in browser console like 
```
Warning: Failed prop type: LineLayer: prop type `dataComparator` is invalid; it must be a function, usually from React.PropTypes.
    in LineLayer (created by Root)
    in Root
  | printWarning | @ | warning.js:35
  | warning | @ | warning.js:59
  | checkReactTypeSpec | @ | checkReactTypeSpec.js:80
  | validatePropTypes | @ | ReactElementValidator.js:162
  | createElement | @ | ReactElementValidator.js:216
  | render | @ | app.js:38
```
I didn't include all stacktrace, but `app.js:38` pointed to `<LineLayer` and other warnings complained about `<ScatterplotLayer` as well.

The screenshot was taken from Chrome 59 on macOS 10.11

**Comment by ibgreen** [comment]
> warnings in browser console like

Yes this is another bug. It seems by mistake we set a `propTypes` on the base `Layer` and React is trying to interpret these. We can fix this soon, hopefully you can just ignore the warnings for now.

Maybe you can just do a `delete Layer.propTypes` in your application to work around the issue for now?

**Comment by AnotherStop** [comment]
no worries, I'm playing with 3d-heatmap example for now. It works for me.

**Comment by ibgreen** [comment]
This is still not fixed. I had labeled it as confirmed bug with a patch milestone attached.

**Comment by 1chandu** [comment]
This should be fixed by react-map-gl 3.0.5 patch
